.github/workflows/3x.yml

```diff
@@ -76,6 +76,7 @@ jobs:
       TERMINUS_VERIFY_HOST_CERT: ${{ vars.TERMINUS_VERIFY_HOST_CERT }}
       HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK: 1
       HOMBEBREW_NO_AUTO_UPDATE: 1
+      TERMINUS_ALLOW_UNSUPPORTED_NEWER_PHP: 1
```

Comment:
Don't set this in CI, update wherever those supported values are set.

Reply:
My intention was just to surface errors in this PR to address them later. But I've failed to do so 😂  I guess we will wait until we actually prioritize a ticket to provide this support :)